### ISSUE TITLE
[Footer/Focus state] focus state color contrast may be failing on dark colors in Safari

### ISSUE THREAD

**Issue by mollieru** [18F/web-design-standards#537]
Concerned that link focus state on darker colors may not be passing accessibility contrast ratio. 

Saw this on Footer in Safari
![screen shot 2015-09-15 at 3 06 15 pm](https://cloud.githubusercontent.com/assets/3680494/9887043/8eb5be7c-5bbb-11e5-8907-6870c1ccb99b.png)

*Tested on*
OS 10.8
Safari 6.2

Note: may be out of scope to resolve for MVP

**Comment by carodew** [comment]
Based on previous conversations with Bristow, we think this will pass WCAG standards. We'd still like to address it but probably after MVP.

**Comment by maya** [comment]
Answered above.
